README.md

```diff
@@ -70,13 +70,51 @@ Then simply inspect the results (`git status`, `git diff`, etc.) and commit.
 
 ## Navigation & Sidebar
 
-The docs (currently) are split into 4 sections:
+The docs (currently) are split into 3 main sections:
 
-- what
```

Comment:
I actually have some new material planned for this section in the coming weeks :smile: